sdk/core/Microsoft.Extensions.Azure/src/Microsoft.Extensions.Azure.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <Description>Azure Client SDK integration with Microsoft.Extensions libraries</Description>
     <AssemblyTitle>Azure Client SDK integration Microsoft.Extensions</AssemblyTitle>
-    <VersionPrefix>1.0.0</VersionPrefix>
+    <VersionPrefix>1.1.0-preview.1</VersionPrefix>
```

Comment:
Could you change this property to `Version` instead of `VersionPrefix` while you are in here?